src/client.js

```diff
@@ -0,0 +1,46 @@
+/* Client factory.
+ *
+ * Generate an OpenAPI client for a spec.
+ */
+import { set } from 'lodash';
+
+import { operationNameFor } from './naming';
+import CallableOperation from './operation';
+
+
+/* Generate a mapping from (dotted) operation names to callable operations.
+ */
+export default (spec, serviceName, options) => {
```

Comment:
nit: `export default function(spec, name = null, options = {})`

Reply:
because?

Reply:
* default options are nice (and you use them in `src/index.js`)
* function names are useful for stack traces